ui/app/AppMain.qml

```diff
@@ -617,6 +617,7 @@ Item {
             id: sendModal
             active: false
             
+            property var selectedAccount
```

Comment:
not sure I have a strong preference here, but could this be a function instead? just to follow with open/close that are already implemented this way.

Reply:
you mean a setter function such as setSelectedAccount? but then again I need to store it somewhere cause only after the item is loaded can I assign the value to the SendModal Component.